packages/echo-base/src/errors/BaseError.test.ts

```diff
@@ -0,0 +1,68 @@
+import { BaseError, getAllProperties } from './BaseError';
+
+describe('BaseError', () => {
+    const message = 'this is a test';
+    it('should initialize with correct values', () => {
```

Comment:
Consistency: it would be nice to have an extra line before this it() clause, to separate the tests - making this more readable (and following the pattern in this file)